.travis.yml

```
@@ -73,6 +84,70 @@ install:
     # section if they are needed before setting up conda) to install any
     # other dependencies.
 
+matrix:
+
+    # Don't wait for allowed failures
+    fast_finish: true
+
+    include:
+        # Make sure that egg_info works without dependencies
+        - stage: Initial tests
+          env: PYTHON_VERSION=3.7 SETUP_CMD='egg_info'
+
+        # Try MacOS X, usually enough only to run from cron as hardly there are
+        # issues that are not picked up by a linux worker
+        - os: osx
+          stage: Cron tests
+          env: SETUP_CMD='test' EVENT_TYPE='cron'
+
+        # Do a coverage test.
+        - os: linux
+          stage: Initial tests
+          env: SETUP_CMD='test --coverage'
+
+        # Check for sphinx doc build warnings - we do this first because it
+        # may run for a long time
+        - os: linux
+          env: SETUP_CMD='build_docs -w'
+               CONDA_DEPENDENCIES=$CONDA_DEPENDENCIES_DOC
```

I'm not at all sure why this line is ignored. Try pip install instead.
```suggestion
               PIP_DEPENDENCIES="$PIP_DEPENDENCIES sphinx-astropy"
```